src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -232,36 +278,76 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
+                      <Checkbox
+                        checked={fileHasFunderRrs}
+                        onChange={() => {
+                          if (fileHasFunderRrs) {
+                            const nullRrsValue: FileRrs = {
+                              type: 'NullRightsRetentionStrategy',
+                              configuredType: rrsStrategy,
+                            };
+                            setFieldValue(rrsFieldName, nullRrsValue);
+                            setFieldValue(licenseFieldName, null);
+                          } else {
+                            const newRrsValue: FileRrs = {
+                              type: 'FunderRightsRetentionStrategy',
+                              configuredType: rrsStrategy,
+                            };
+                            setFieldValue(rrsFieldName, newRrsValue);
+                            setFieldValue(licenseFieldName, LicenseUri.CC_BY_4);
+                          }
+                        }}
+                      />
+                    }
+                  />
+                )}
+
+                {fileHasCustomerRrs && isCustomerRrs && (
+                  <Typography>
+                    {t('registration.files_and_license.institution_rights_policy_opt_out_instructions')}
+                  </Typography>
+                )}
+
+                {canOverrideRrs && (
+                  <FormControlLabel
+                    label={
+                      <Trans t={t} i18nKey="registration.files_and_license.follow_institution_rights_policy">
+                        {rrsPolicyLink}
+                      </Trans>
+                    }
+                    control={
+                      <Checkbox
+                        checked={!fileHasOverriddenRrs}
+                        onChange={() => {
+                          if (fileHasOverriddenRrs) {
+                            const customerRrsValue: FileRrs = {
+                              type: 'CustomerRightsRetentionStrategy',
+                              configuredType: rrsStrategy,
+                            };
+                            setFieldValue(rrsFieldName, customerRrsValue);
+                            setFieldValue(licenseFieldName, LicenseUri.CC_BY_4);
+                          } else {
+                            const overriddenRrsValue: FileRrs = {
+                              type: 'OverriddenRightsRetentionStrategy',
+                              configuredType: rrsStrategy,
+                            };
+                            setFieldValue(rrsFieldName, overriddenRrsValue);
+                            setFieldValue(licenseFieldName, null);
+                          }
+                        }}
+                      />
+                    }
+                  />
+                )}
 
-                {user?.isPublishingCurator && (
+                {(user?.isPublishingCurator || (isOverridableRrs && fileHasOverriddenRrs)) && (
```

Comment:
PO har rapportert en bug, tror dette skal vekk (Dobbeltsjekker før jeg tar vekk)
```suggestion
                {user?.isPublishingCurator && (
```